challenges/classes.js

```diff
@@ -1,7 +1,29 @@
+console.log('==========================Classes==========================')
+
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMakerRefactor {
+    constructor(stuff) {
+        this.length = stuff.length;
+        this.width = stuff.width;
+        this.height = stuff.height;
```

Comment:
That's a lot of stuff. A good way to also refactor this one is to name each separate callback. Still though, great work!